systemtests/src/main/java/io/enmasse/systemtest/messagingclients/AbstractClient.java

```diff
@@ -105,6 +107,14 @@ public JsonArray getMessages() {
         return messages;
     }
 
+    /**
+     * Get client capability such as sender/receiver.
```

Comment:
I wonder, would 'role' be a better term than capability. When seeing 'capability' i think about AMQP source/target capabilities that you can set like 'queue' or 'topic'. I think in the future we may want to be able to pass this on to the client as well, so perhaps rename this now. The term 'role' is used throughout the AMQP 1.0 spec.

Reply:
Okay, that sounds reasonable. Going to change it.